coalib/bears/requirements/JuliaRequirement.py

```diff
@@ -29,3 +29,22 @@ def __init__(self, package, version="", flag=""):
         """
         PackageRequirement.__init__(self, 'julia', package, version)
         self.flag = flag
+
+    def install_command(self):
+        """
+       Creates the installation command for the instance of the class.
+
+       >>> JuliaRequirement('("Pkg.add(\"Lint\")', '' , '-u' ).install_command()
+       'julia -e "Pkg.add(\"Lint\")"'
+
+       :param return: A string with the installation command.
+       """
+        return "julia get {} {}".format(self.flag, self.package)
+
+    def is_installed(self):
+        """
+       Checks if the dependency is installed.
+
+       :param return: True if dependency is installed, false otherwise.
+       """
+        return not call_without_output(('julia', 'show', self.package),)
```

Comment:
i dont think ``julia show`` exists as a command, you need to search on the internet how to check if a julia package is installed, they may have some kind of method.